test/blackbox-tests/test-cases/coq/coqtop.t/run.t

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
This test needs a documentation header. A quick description of what it is being test would suffice.